### DISCUSSION TITLE
Slider onChange callback can't see states of calling component

### DISCUSSION THREAD

**Discussion by ginfonic** [mantinedev/mantine#3251] [Q&A]
In Slider component when you use a custom callback in onChange prop that not only set a value but also uses internal states of calling component these states are invisible, undefined or 0.
The other Mantine components, Select for instance or the standard ones (input) can see calling component states in callback normally as it should be.
Maybe I do something wrong?

<img width="851" alt="Снимок экрана 2022-12-28 в 11 36 12" src="https://user-images.githubusercontent.com/537747/209783500-54ccccc1-7eea-4a77-b4e6-65eaeaded0f9.png">

<img width="864" alt="log" src="https://user-images.githubusercontent.com/537747/209782911-cdad06ac-4a4f-4a0c-95f0-cbf1b83d474c.png">

**Comment by rtivital** [comment]
Please do not duplicate your questions both to issues and discussions
